### DISCUSSION TITLE
Welcome to Score Community Discussions!

### DISCUSSION THREAD

**Discussion by gguizza** [score-spec/spec#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Participate to community meetings.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kimnology** [comment]
Greetings, I am new to score and seeking to learn how to use and configure the software.

**Reply by gguizza** [reply]
Hey! great to have you here :)
To start, please check our get started list at this link: [Get Started Guide](https://score.dev/get-started)
then our documentation and our tutorials here: [Documentation](https://docs.score.dev/docs/) & [Tutorial](https://score.dev/blog/get-started-with-score-docker-compose-to-helm)
Let me know if you have any other question!
